bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-participants/list-item/component.tsx

```diff
@@ -184,11 +184,17 @@ const UserListItem: React.FC<UserListItemProps> = ({ user, lockSettings }) => {
     } return '';
   };
 
+  // This is being done because destructing is not a real solution, isDialIn is just a variable name.
+  // eslint-disable-next-line prefer-destructuring
+  const isDialIn = user.isDialIn;
+
   const iconUser = getIconUser();
+  const iconDialInUser = (<Icon iconName="volume_level_2" />);
+  const userIcon = isDialIn ? iconDialInUser : iconUser;
```

Comment:
It it possible to use the function `getIconUser` to validate whether the user is `isDialIn` and return the correct icon?
It seems it will be more organized to keep all the logic inside a single function.